### ISSUE TITLE
qTox is changing my password profile!

### ISSUE THREAD

**Issue by qtoxer** [qTox/qTox#3050]
Using Linux Mint 17.3 Operating System
qTox version: the latest. 
**Steps of the issue:**  
1.  Did the latest update today which is v1.3.0-80-g252c9c8
2.  After rebooting the system, and logged in.
3.  Used the same password. It shows "Wrong Password"
…

**Additional info:**  
![3340ua8](https://cloud.githubusercontent.com/assets/3148759/14022397/8a152486-f1d7-11e5-8868-ec0332bb238b.jpg)

_edited by zetok – removed link to image and included image in the issue_

**Comment by zetok** [comment]
Could it be #2161?

Edit: i.e. are you sure that you had free space on your HDD?

**Comment by qtoxer** [comment]
I've checked it, and there is a lot of space. Also I've tried to use the command : 
`qtox -p profile`
But it does not delete the profile, and I could not load any other profiles using the mouse nor the keyboard.
Do I have to uninstall the app, or what should I do?

**Comment by zetok** [comment]
The size of your `profile.tox` file is not `0`, right?

Edit:

> Do I have to uninstall the app, or what should I do?

I hope that you have backups..

**Comment by qtoxer** [comment]
> The size of your profile.tox file is not 0, right?

That is right, the profile.tox file is not 0 KB.

> I hope that you have backups..

Yes I did, but is the uninstallment of qTox with reinstalling it a solution? or there is a better way?

**Comment by zetok** [comment]
> Yes I did, but is the uninstallment of qTox with reinstalling it a solution? or there is a better way?

Reinstalling won't do anything – your profile is encrypted, and most likely was somehow corrupted, possibly by qTox, which would lead to a situation where qTox is not able to decrypt a corrupted file.

What you would want to do, is to save that file somewhere, try to restore file that you know was working from a backup, and see if that works.

**Comment by qtoxer** [comment]
> try to restore file that you know was working from a backup, and see if that works.

Sorry, but I'm not an expert on this. Do you mean restore the profile.tox file to`~/.config/tox/` ?

**Comment by qtoxer** [comment]
Here is what happened before, and after the issue:
Usually I use LXDE desktop environment, and since it shows the update manager with accessing the root too it was easy to update and manage files around; unlike Enlightenment desktop environment it does not allow to use root nor any app that may want root access. Although, Enlightenment desktop environment does take less RAM than LXDE desktop environment.
https://l3net.files.wordpress.com/2014/02/cmp-all4.png
I've been using Enlightenment desktop environment for few weeks, but after doing the latest update of qTox on LXDE. I logged out from LXDE desktop environment, and signed in with Enlightenment desktop environment. On the list of the start-up applications is qTox, and I've logged to the profile normally. After leaving the PC on for few hours, the Enlightenment desktop environment got laggy so I had to reboot the system. Then the issue start to happened, and I could not access to the profile.
